lambda-udfs/f_glue_schema_registry_avro_to_json(varchar,varchar)/lambda.yaml

```diff
@@ -0,0 +1,150 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  SecurityGroupId:
+    Description: the SG this function should run in to have connectivity to Redshift
+    Type: String
+  SubnetId:
+    Description: the Subnet this function should run in to have connectivity to Redshift
+    Type: String
+  S3Bucket:
+    Description: the S3 Bucket containing the Lambda layer
+    Type: String
+  S3Key:
+    Description: the S3 Key containing the Lambda layer
+    Type: String
+Resources:
+  LambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+          Version: 2012-10-17
+          Statement:
+            -
+              Effect: Allow
+              Principal:
+                Service:
+                  - lambda.amazonaws.com
+              Action:
+                - sts:AssumeRole
+      Path: /
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
+      Policies:
+          -
+            PolicyName: CloudwatchLogs
+            PolicyDocument:
+              Version: 2012-10-17
+              Statement:
+                -
+                  Effect: Allow
+                  Action:
+                    - logs:CreateLogGroup
+                  Resource:
+                    - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:*"
+                -
+                  Effect: Allow
+                  Action:
+                    - logs:CreateLogStream
+                    - logs:PutLogEvents
+                  Resource:
+                    - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/*"
+                -
+                  Effect: Allow
+                  Action:
+                    - glue:GetSchemaVersion
+                  Resource:
+                    - !Sub "arn:${AWS::Partition}:glue:${AWS::Region}:${AWS::AccountId}:registry/*"
+                    - !Sub "arn:${AWS::Partition}:glue:${AWS::Region}:${AWS::AccountId}:schema/*/*"
+  AvroLayer:
+    Type: AWS::Lambda::LayerVersion
+    Properties: 
+      CompatibleRuntimes: 
+        - python3.9
+      Content: 
+        S3Bucket: !Ref S3Bucket
+        S3Key: !Sub "${S3Key}/avro_1.11.2.zip"
+      Description: Python 3.9 Avro library
+      LayerName: avro
+  LambdaUDFFunction:
+     Type: "AWS::Lambda::Function"
+     Properties:
+       FunctionName: f-glue-schema-registry-avro-to-json-varchar-varchar
+       Role: !GetAtt 'LambdaRole.Arn'
+       Handler: index.handler
+       Runtime: python3.9
+       Timeout: 300
+       Layers:
+        - !Ref AvroLayer
+       VpcConfig:
+        SecurityGroupIds:
+          - !Ref SecurityGroupId
+        SubnetIds:
+          - !Ref SubnetId
+       Environment:
+        Variables:
+          GLUE_REGISTRY_NAME: AvroLambdaUDF
```

Comment:
Instead of hard-coding this, you should add as a parameter e.g. f-glue-schema-registry-avro-to-json-varchar-varchar-varchar